scipy/stats/morestats.py

```diff
@@ -1215,28 +1253,337 @@ def boxcox_normplot(x, la, lb, plot=None, N=80):
     >>> plt.show()
 
     """
+    return _normplot('boxcox', x, la, lb, plot, N)
+
+
+def yeojohnson(x, lmbda=None):
+    r"""
+    Return a dataset transformed by a Yeo-Johnson power transformation.
+
+    Parameters
+    ----------
+    x : ndarray
+        Input array.  Should be 1-dimensional.
+    lmbda : {None, scalar}, optional
+        If `lmbda` is not None, do the transformation for that value.
```

Comment:
These paragraphs can be combined. Note that you need double backticks for code formatting. 
```
If ``lmbda`` is None, find the lambda that maximizes the log-likelihood function and return it as the second output argument. Otherwise the transformation is done for the given value.
```